nwbinspector/checks/nwbfile_metadata.py

```diff
@@ -127,6 +127,18 @@ def check_subject_age(subject: Subject):
         )
 
 
+@register_check(importance=Importance.BEST_PRACTICE_SUGGESTION, neurodata_type=Subject)
+def check_subject_age_reference(subject: Subject):
+    """Check if the Subject age reference is one of supported options."""
+    valid_options = ["birth", "?"]
+    if subject.age.reference not in valid_options:
+        return InspectorMessage(
+            message=(
+                f"Subject age reference, '{subject.age.reference}', is not one of the valid options ({valid_options})."
```

Comment:
```suggestion
    if subject.age__reference not in valid_options:
        return InspectorMessage(
            message=(
                f"Subject age reference, '{subject.age__reference}', is not one of the valid options ({valid_options})."
```

Reply:
@CodyCBakerPhD when there is a neurodata_type with a dataset that has an attribute, the default mapping is [dset_name]__[attr_name]